__docs__/wonder-blocks-labeled-field/accessibility.mdx

```diff
@@ -25,3 +25,84 @@ prop is not provided, a unique id will be auto-generated!
 - The `LabeledField` component does not need to be used with the `CheckboxGroup`
  or `RadioGroup` components since those components already have accessible labels
  built-in when the `label` prop is used.
+
```

Comment:
Elaborating more around the screen reader behaviour from a previous PR comment: https://github.com/Khan/wonder-blocks/pull/2344#discussion_r1797493530